lms/templates/verify_student/pay_and_verify.html

```diff
@@ -65,7 +65,6 @@
       data-platform-name='${platform_name}'
       data-course-key='${course_key}'
       data-course-name='${course.display_name}'
-      data-course-start-date='${course.start_datetime_text()}'
```

Comment:
It's a formatted date/time string, like "Dec 1, 2016", and I couldn't find it in my searching. I *hope* it's not still in use, then we've got us a dead PR.